pkgs/by-name/sp/spade/package.nix

```
@@ -39,14 +41,19 @@ rustPlatform.buildRustPackage rec {
     '')
   ];
 
+  env.NIX_CFLAGS_LINK = lib.optionals stdenv.isDarwin "-L${python312}/lib/python3.12/config-3.12-darwin -lpython3.12";
```

I think this also needs the buildInputs include:
```suggestion
  buildInputs = lib.optionals stdenv.isDarwin [ python312 ];
  env.NIX_CFLAGS_LINK = lib.optionals stdenv.isDarwin "-L${python312}/lib/python3.12/config-3.12-darwin -lpython3.12";
```